eloquentjs/16_platform_game_ts/src/interfaces.ts

```diff
@@ -0,0 +1,40 @@
+interface Vector2 {
```

Comment:
It's generally better to put the type definitions for things like Vector2 in the file that implements them because it makes go to definition easier.